modules/installation-guide/partials/assembly_building-offline-registry-images.adoc

```diff
@@ -0,0 +1,7 @@
+[id="building-offline-registry-images_{context}"]
+= Building offline registry images
+
+include::partial$proc_building-an-offline-devfile-registry-image.adoc[leveloffset=+1]
```

Comment:
When I deploy this, I think something is missing. The build steps for the offline plugin and devfile registries are the same, whereas one should be using NodeJS/yarn and the other is just running the `build.sh` file with no NodeJS/yarn prerequisite.

This is especially odd given the comment I left below in `modules/installation-guide/partials/proc_building-an-offline-devfile-registry-image.adoc`

Reply:
I removed the nodejs/yarm prerequisites. Is there still something to change?

Reply:
Yep it's gone :+1: